tools/apigee-x-trial-provision/apigee-x-trial-provision.sh

```diff
@@ -75,6 +83,17 @@ fi
 # Step 1: Define functions and environment variables
 function token { echo -n "$(gcloud config config-helper --force-auth-refresh | grep access_token | grep -o -E '[^ ]+$')" ; }
 
+function gcloud_ignore_exist {
```

Comment:
i understand what you are doing, but i think it is simpler to understand...

`gcloud create something -q || echo "Couldn't create - is it already created?"`

this reduces the LOC of this script and makes it clear that it might fail for a different reason, but the most likely is that it exists.

alternatively - it would also be nicer to do 

```
if ! gcloud get something ; then
  gcloud create something
fi
```

rather than this error handling which is a little hard to understand. in both cases you also dont have to jump around the file to understand whats going on so it is nicer to demo